src/App.tsx

```diff
@@ -15,71 +15,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGoods: string | null,
```

Comment:
```suggestion
  selectedGood: string | null,
```
and in which case it will be null?